Tests/ParseSwiftTests/ACLTests.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Corey Baker on 8/22/20.
 //  Copyright © 2020 Parse Community. All rights reserved.
 //
-
+#if !os(watchOS)
```

Comment:
Is this necessary to include in the tests? Are we running tests separately just for the Watch?

Reply:
From what I've been finding online, XCTest doesn't run on watchOS. Currently, we don't get the watchOS badge on the readme because watchOS can't compile any of the tests. I used a solution discussed on the swift repo which use `!os(watchOS)`, there's been some discussions about this, doesn't seem like there's a preferred way to go around this.

With the above flag, the readme should auto populate the watchOS badge, assuming the build goes fine

Reply:
Sure, that makes sense.